src/main/java/com/google/opensesame/servlets/MentorsServlet.java

```diff
@@ -26,11 +26,13 @@
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response)
       throws ServletException, IOException {
+    addMockMentor(response); // TODO: remove by production
     List<UserEntity> mentorEntities = ofy().load().type(UserEntity.class).list();
```

Comment:
IIUC this list doesn't only contain mentors (otherwise the `if (entity.isMentor())` statement is unnecessary. Rename `mentorEntities` to something else that conveys that this list also contains people who aren't mentors. A suggestion I have is to call it `userEntities`.